sdk/communication/Azure.Communication.CallAutomation/src/Models/Events/CallAutomationEventParser.cs

```diff
@@ -112,10 +112,14 @@ private static CallAutomationEventBase Deserialize(string eventData, string type
                     return PlayCompleted.Deserialize(eventData);
                 case nameof(PlayFailed):
                     return PlayFailed.Deserialize(eventData);
+                case nameof(PlayCanceled):
+                    return PlayCanceled.Deserialize(eventData);
                 case nameof(RecognizeCompleted):
                     return RecognizeCompleted.Deserialize(eventData);
                 case nameof(RecognizeFailed):
                     return RecognizeFailed.Deserialize(eventData);
+                case nameof(RecognizeCanceled):
```

Comment:
Lets keep uniformity for "Canceled" spelling. Check the comment on the other PR for context. "fmorales/event_enum"